### ISSUE TITLE
Support for debug console

### ISSUE THREAD

**Issue by FWest98** [FlorentRevest/vscode-addr2line#1]
Thanks the the plugin! Do you think it'd be possible to adapt the plugin to also support the debug console output? I typically print my dmesg output using `-exec lx-dmesg` in a GDB session, and currently this usecase seems to not be covered

**Comment by FlorentRevest** [comment]
Unfortunately this does not seem to be possible no :/ https://github.com/microsoft/vscode/issues/210428

**Comment by FWest98** [comment]
Ah, that's unfortunate. Maybe as a more general alternative, do you think it would be possible to make a command palette entry that allows me to copy/paste an address?

**Comment by FlorentRevest** [comment]
Yes that would actually be possible, and quite straightforward to implement. Is this something you'd be interested in contributing ? I don't think I'd be using it so I don't have an urge to implement it but if you send me a PR I would happily review it.
